packages/orbit-components/.storybook/orbitDecorator.js

```diff
@@ -1,29 +1,29 @@
-import * as React from "react"
-import Heading from "../src/Heading"
-import Text from "../src/Text"
-import jsxToString from "react-element-to-jsx-string"
-import { Code } from "@storybook/addon-info/dist/components/markdown/"
+import * as React from "react";
+import Heading from "../src/Heading";
+import Text from "../src/Text";
+import jsxToString from "react-element-to-jsx-string";
+import { Code } from "@storybook/addon-info/dist/components/markdown/";
 
 const orbitDecorator = (storyFn, context) => {
-  const children = storyFn(context)
+  const children = storyFn(context);
   const options = {
     filterProps: val => val != null && val !== "",
     functionValue: () => {
       return "function()";
     },
   };
-  return  (
+  return (
     <div style={{ padding: "20px" }}>
       <Heading spaceAfter="medium">{context.kind}</Heading>
       <Text spaceAfter="largest">{context.parameters?.info}</Text>
       {children}
-      <div style={{ marginTop: 20}}>
+      <div style={{ marginTop: 20 }}>
         <React.Fragment>
           <Code code={jsxToString(children, options)} language="jsx" format={false} />
         </React.Fragment>
       </div>
     </div>
-  )
-}
+  );
+};
```

Comment:
I was wondering how this got past ESLint. It's because ESLint by default ignores all paths starting with `.`, which includes `.storybook`, so please also add `!.storybook` to `.eslintignore`.

Reply:
thanks, did not notice that